stdlib/ballerina-http/findbugs-exclude.xml

```diff
@@ -33,7 +33,7 @@
         <Class name="org.ballerinalang.net.uri.parser.SimpleSplitStringExpression"/>
     </Match>
     <Match>
-        <Class name="org.ballerinalang.net.ws.Constants"/>
+        <Class name="org.ballerinalang.net.ws.WebSocketConstants"/>
```

Comment:
@irunika why do we exclude this here ?

Reply:
This has been done in the old code since find bugs find that extending Constants from Constants is a bug. 
@riyafa can we remove this and remove the extended HttpConstant class from it since now we can use it without using full package path in the code it self?

Reply:
Since the `WebSocketConstants` do not extent `HttpConstant` I have removed this exclusion now.